src/pages/workspaces/workspace/workflows/WorkflowView.test.js

```diff
@@ -0,0 +1,126 @@
+import { act, screen } from '@testing-library/react';
+import { h } from 'react-hyperscript-helpers';
+import { Ajax } from 'src/libs/ajax';
+import { WorkflowView } from 'src/pages/workspaces/workspace/workflows/WorkflowView';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+
+jest.mock('src/libs/ajax');
+jest.mock('src/libs/nav', () => ({
+  ...jest.requireActual('src/libs/nav'),
+  getLink: jest.fn().mockImplementation((_) => _),
+}));
+
+describe('Workflow View', () => {
+  it('imports successfully', async () => {
```

Comment:
Is this testing workflow import? Isn't this component viewing a workflow that's already added to a workspace?

Reply:
That's a better name for it. In the integration test, this was testing importing a workflow from the methods repo, to Terra, but since I'm mocking the methods repo response, there's probably a better name. Maybe 'view workflow in workspace from mock import'